app/components/theme/switcher_component.yml

```diff
@@ -1,5 +1,5 @@
 link:
-  base: 'text-gray-700 group flex items-center text-sm dark:text-gray-300'
+  base: 'text-gray-600 group flex items-center dark:text-gray-300'
```

Comment:
Nit: This change appears to be in the codebase already. Did a commit from an earlier PR make it into this one?